addons/l10n_ch/data/account_tax_report_data_2024.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Tax Report (post-2024 rates change) -->
+    <record id="tax_report_2024" model="account.tax.report">
+        <field name="name">Tax Report (2024)</field>
+        <field name="country_id" ref="base.ch"/>
+    </record>
+
+    <record id="account_tax_report_line_chiffre_af_2024" model="account.tax.report.line">
+        <field name="name">I - CHIFFRE D'AFFAIRES</field>
+        <field name="report_id" ref="tax_report_2024"/>
+        <field name="formula">None</field>
+        <field name="sequence" eval="1"/>
+    </record>
+
+    <record id="account_tax_report_line_chtax_200_2024" model="account.tax.report.line">
+        <field name="name">200 Chiffre d'affaires</field>
+        <field name="formula">tax_ch_302a_2024 + tax_ch_312a_2024 + tax_ch_342a_2024 + tax_ch_289_2024</field>
+        <field name="report_id" ref="tax_report_2024"/>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="account_tax_report_line_chiffre_af_2024"/>
+    </record>
+
+    <record id="account_tax_report_line_chtax_289_2024" model="account.tax.report.line">
+        <field name="name">289 Déductions (ch.220 à ch.280)</field>
+        <field name="report_id" ref="tax_report_2024"/>
+        <field name="sequence" eval="2"/>
+        <field name="code">tax_ch_289_2024</field>
+        <field name="parent_id" ref="account_tax_report_line_chiffre_af_2024"/>
+    </record>
+
+    <record id="account_tax_report_line_chtax_220_289_2024" model="account.tax.report.line">
+        <field name="name">220 Chiffre d'affaires imposable a 0% (export)</field>
+        <field name="tag_name">220_2024</field>
+        <field name="report_id" ref="tax_report_2024"/>
+        <field name="sequence" eval="0"/>
+        <field name="parent_id" ref="account_tax_report_line_chtax_289_2024"/>
+    </record>
+
+    <record id="account_tax_report_line_chtax_221_2024" model="account.tax.report.line">
+        <field name="name">221 Prestations fournies à l'étranger</field>
+        <field name="tag_name">221_2024</field>
```

Comment:
Shouldn't we reuse the same tags instead of adding a _2024 suffix and essentially duplicating them ?